response/slack/views.py

```diff
@@ -70,6 +70,10 @@ def slash_command(request):
                 name="severity",
                 optional=True,
             ),
+            SelectWithOptions(
```

Comment:
Based on the the comment about this being optional functionality, could we add this block conditionally based on the `INCIDENT_REPORT_CHANNEL_ID` being set?  We'd also need to modify the dialog handler so it handles it not being present.

Reply:
Sure, that seems like the simplest way to make this a configurable feature 👍